nautobot/extras/tests/test_notes.py

```diff
@@ -0,0 +1,51 @@
+from django.contrib.auth import get_user_model
+from django.contrib.contenttypes.models import ContentType
+from django.db.models.query import QuerySet
+
+from nautobot.dcim.models import Site
+from nautobot.extras.models import Note
+from nautobot.utilities.testing import TestCase
+
+
+User = get_user_model()
+
+
+class NoteModelTest(TestCase):
+    @classmethod
+    def setUpTestData(cls):
+
+        site_ct = ContentType.objects.get_for_model(Site)
+
+        user = User.objects.first()
+
+        cls.sites = [
+            Site.objects.create(name="Site 1", slug="site-1"),
+            Site.objects.create(name="Site 2", slug="site-2"),
+        ]
+
+        Note.objects.create(
+            name="Site Maintenance",
```

Comment:
I believe the test is failing because `name` is no longer a valid parameter here and below.

Reply:
Yeah, I thought I fixed this everywhere, but turns out I missed one. Thank you!